examples/tracing/python/README.md

```diff
@@ -0,0 +1,57 @@
+# Span Profiles with Grafana Tempo and Pyroscope
```

Comment:
I think the instructions also said we should link back and forth between https://grafana.com/docs/pyroscope/latest/configure-client/trace-span-profiles/python-span-profiles/ and this README.md